pkg/schema/openapi.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/k14s/ytt/pkg/yamlmeta"
 )
 
+// keys used when generating an OpenAPI Document
+const (
+	Type        = "type"
+	Default     = "default"
+	Nullable    = "nullable"
+	Description = "description"
```

Comment:
```suggestion
	typeProp        = "type"
	defaultProp     = "default"
	nullableProp    = "nullable"
	descriptionProp = "description"
```

Suggestions:
- un-export these constants as they are not intended to be used outside the package.
- need to disambiguate `type` from the Go keyword... for example add the `Prop` suffix. 🤷🏻